client/stylesheets/shared/_global.scss

```diff
@@ -110,4 +115,29 @@
 		margin-top: 4px;
 		background-color: $studio-white;
 	}
+
+	.muriel-input-text.empty input {
+		top: 12px;
+	}
+
+	.muriel-input-text.with-value input {
+		top: 24px;
+	}
+
+	.muriel-checkbox .components-base-control__field {
+		align-items: center;
+		display: flex;
+	}
+}
+
+.wp-version-less-than-5-3 {
```

Comment:
Instead of introducing a new class, could we update the property that caused the break?

For example, it looks like adding `min-height: auto;` to the muriel input tags will fix the positioning and overlap seen in the images.

```css
.muriel-input-text input {
    min-height: auto;
}
```